nexus/social_media/services.py

```diff
@@ -0,0 +1,30 @@
+# Third Party Stuff
+import facebook
+from nexus.celery import app
```

Comment:
Shouldn't this be present under `Nexus Stuff`?

Reply:
Please use `pre-commit-hooks` that are already in the repo. It would make sure you are able to commit better quality code.

Use `pre-commit` install, and then try to commit. It would also try to fix some minor things, and suggest you fix other things yourself. Should be pretty handy, to avoid back-and-forth reviews for minor stuff :)

cc @GeekyShacklebolt @storymode7